### DISCUSSION TITLE
How to use in nuxt 3?

### DISCUSSION THREAD

**Discussion by JenuelDev** [ismail9k/vue3-carousel#271] [Q&A]
How to use in nuxt 3?

**Comment by CodeMarcin** [comment]
Same q.

**Comment by ismail9k** [comment]
Here is a working [example](https://stackblitz.com/edit/github-1vtnku?file=components/BasicCarousel.vue) with Nuxt@3.0.0

**Reply by oldravian** [reply]
it says 404 no found

**Reply by ismail9k** [reply]
You can check this nuxt package: https://github.com/gaetansenn/vue3-carousel-nuxt
